src/node_buffer.cc

```diff
@@ -520,6 +520,22 @@ MaybeLocal<Object> New(Environment* env,
 
 namespace {
 
+// TODO: Buffer.from(buf, fromEnc).toString(buf, toEnc)
+void BufferToString(const FunctionCallbackInfo<Value>& args) {
+    // ArrayBufferViewContents<char> buffer(args[0]); // Error
+    CHECK(args[0]->IsString());
```

Comment:
This is wrong. This should be:

```suggestion
    CHECK(args[0]->IsArrayBuffer() || args[0]->IsSharedArrayBuffer() || args[0]->IsArrayBufferView());
```